transactor/src/test/scala/io/mediachain/transactor/CborSerializationSpec.scala

```diff
@@ -43,12 +55,59 @@ object CborSerializationSpec extends BaseSpec {
       meta = Map("created" -> CString("the past"))
     )
 
+    val entityUpdateCell = EntityUpdateCell(
+      entity = multihashRef("foo"),
+      chain = Some(multihashRef("bar")),
+      meta = Map("favoriteColor" -> CString("Blue! No, Yellow!!!"))
```

Comment:
it's really fun to come up with these but I would stick them into a list and pull one out at random for each test, similar to how we do it in the L-SPACE fixtures https://github.com/mediachain/L-SPACE/blob/master/l_space/src/test/scala/io/mediachain/GraphFixture.scala#L47

I would also inject some unicode in here